nuxeo-drive-client/tests/test_synchronization.py

```diff
@@ -7,12 +7,19 @@
 from nxdrive.client.remote_filtered_file_system_client import \
     RemoteFilteredFileSystemClient
 from nxdrive.osi import AbstractOSIntegration
-from tests import RemoteTestClient
-from tests.common import OS_STAT_MTIME_RESOLUTION, TEST_WORKSPACE_PATH
-from tests.common_unit_test import DEFAULT_WAIT_SYNC_TIMEOUT, RandomBug, \
+from . import RemoteTestClient
+from .common import OS_STAT_MTIME_RESOLUTION, TEST_WORKSPACE_PATH
+from .common_unit_test import DEFAULT_WAIT_SYNC_TIMEOUT, RandomBug, \
```

Comment:
On the Python Client, I used parentheses around lists of imports to avoid the \ when continuing on another line. Should we keep the same convention? Just a cosmetic question.